BL/ProjectManager.cs

```diff
@@ -114,7 +139,10 @@ public void RemovePhase(int projectId, int phaseId)
                 alteredModule.Phases.Remove(removedPhase);
                 ModuleMan.EditModule(alteredModule);
             }
-            ProjectRepo.Delete(phaseId);
+
+            //TODO: Enkel de phase moet verwijdert worden project niet? 
```

Comment:
Inderdaad. Voor IPT13-28 (Niet voor deze feature) is het best dat je deze methode herschrijft.